src/utils/sendPaginatedOrderList.js

```diff
@@ -0,0 +1,32 @@
+import Order from '../models/order';
```

Comment:
Why to write this new file, if you have `utils/sendPaginatedList`? Why didn't you use the old one?